app/build.gradle

```diff
@@ -251,6 +252,8 @@ android {
         targetSdkVersion 31
         multiDexEnabled true
         applicationId "org.commcare.dalvik"
+        testNamespace "org.commcare.dalvik.test"
```

Comment:
is there a particular reason we need to add this ?

Reply:
Yes, setting _package_ in the manifest file was deprecated in AGP 7.3 and support to package placeholders seems to have been dropped. This causes Android Studio to throw an `Package '${applicationId}' from AndroidManifest.xml is not a valid Java package name as '${applicationId}' is not a valid Java identifier` error when we run the instrumentation tests.

Reply:
makes sense.